examples/Cassie/run_controller_switch.cc

```diff
@@ -1,55 +1,136 @@
 #include <string>
-
 #include <gflags/gflags.h>
+#include "dairlib/lcmt_controller_switch.hpp"
+#include "dairlib/lcmt_robot_output.hpp"
+#include "systems/framework/lcm_driven_loop.h"
```

Comment:
No need for lcm_driven_loop (and maybe other dependencies)

Reply:
Finished cleaning up the include's